woocommerce/woocommerce-android
# AI product description: Description generation bottom sheet

Fixes #9291, a subtask of #9214.

This PR implements the UI of description generation but the there are no network calls being made. 

[device-2023-06-28-175107.webm](https://github.com/woocommerce/woocommerce-android/assets/1522856/83f9c955-d032-4a32-873a-ac915b18f247)

**To test:**
1. Go to Products -> Tap on a product
2. Tap on `Write with AI` button
3. Notice a bottom sheet appears
4. Tap on `Write with AI` button
5. Notice a shimmer view is shown
6. After a moment, a fake description is shown
7. Tap on `Regenerate button`
8. Notice the shimmer view is shown with a progress bar at the bottom
9. The fake description is displayed after a few seconds
